src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +30,9 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber = nextSequenceNumber;
+        nextSequenceNumber++; 
     }
 
```

Comment:
Where is the getter for the private variable
Has the relevant interfaces been updated?